config.yaml

```diff
@@ -25,10 +25,14 @@ http.address: 0.0.0.0:8080
 # HTTP Cache-Control
 #------------------------
 
-cachecontrol.workitemtype: max-age=86400
-cachecontrol.workitemlinktype: max-age=86400
-cachecontrol.space: max-age=300
-cachecontrol.iteration: max-age=300
+cachecontrol.workitemtypes: max-age=86400
```

Comment:
This is looking more closer to a `.properties` file rather than a `.yaml` bc of the properties naming and structure :)